packages/common/src/core/types/DynamicValues.ts

```diff
@@ -0,0 +1,102 @@
+import { ISearchOptions } from "@esri/arcgis-rest-portal";
+import { IConfigurationSchema } from "../schemas";
+
+/**
+ * The source for a dyanmic value. Used to determine how to resolve the value
+ */
+export type DynamicValueSource = "item-query" | "service-query" | "portal";
```

Comment:
```suggestion
export type DynamicValueSource = "item-search" | "service-query" | "portal";
```

Reply:
I'll make thi change and push up vs accepting the commit (I need to also change it in the e2e harness)